### DISCUSSION TITLE
Moving project to new library doesn't bring BRAW video files. Please advise.

### DISCUSSION THREAD

**Discussion by tangierc** [latenitefilms/BRAWToolbox#117] [General]
While testing out performance problems, sync issues, and frame jumps I am having with BRAW Toolbox clips on my timeline I wanted to move a short timeline's clips to my computers internal SSD.  Currently all media is on external hard drives, but otherwise I am using the same computer. 

1. Using Final Cut Pro's copy to Library feature and then consolidating all media into the new library didn't bring any of the BRAW clips.  
2. Clips exist in the timeline, but not the browser and the library still looks for the original external drive locations. 
3. Original, transcoded, and proxies were selected to move over just for good measure.
4. All destinations were set to move to the new library on my computer's internal drive.
5. Only the extracted BRAW clip audio was brought over to the new library in the browser and in the Finder
6. No braw clips are present in the Finder in the new library.

Final Cut Pro 10.6.5 / MacBook Pro M2 Max / Venture macOS 13.3.1

**Comment by latenitefilms** [comment]
Unfortunately the built-in Consolidation tools will never work, because you don't really have the `.braw` files in Final Cut Pro - you just have a "dummy" Generator that has an effect that's "rendering" the BRAW files. It's possible we could add something in the Workflow Extension to workaround this.

**Comment by tangierc** [comment]
Ok, thank you. I figured as much, but I am glad you confirmed. IT seems the only way to move to a new library is through the Finder.  I also noticed today working with the writer/director that stabilization is not an option on the BRAW Toolbox synced clips.  Is this true or is this a bug in my system?

I tried making a synced clip out of a Red proxy and a random audio file and the resulting synchronized clip did allow stabilization.

Thanks,

Tangier

**Reply by latenitefilms** [reply]
See:

https://brawtoolbox.io/stabilisation/

**Comment by tangierc** [comment]
Thanks for pointing me in the right direction.  FCP does allow me to apply stabilization to a synchronized clip that I tested with some other media, but not a multicam clip.

Tangier

**Reply by latenitefilms** [reply]
Good point - yes, Final Cut Pro will allow you to stabilise a Synchronised Clip as long as it just has a standard/supported video file inside it. If you have something non-standard (i.e. two video clips or in the case of BRAW Toolbox, a Generator), then it won't show a stabilisation option.

For example - a Synchronised Clip containing two ProRes Clips: 

<img width="1168" alt="image" src="https://user-images.githubusercontent.com/22286696/233982017-2e7342cd-8ec9-495a-8a5c-c72d23fca85c.png">

A Synchronised Clip containing one ProRes Clip:

<img width="1130" alt="image" src="https://user-images.githubusercontent.com/22286696/233982121-1a25994a-0621-4273-b0b8-65fdf3e88fc9.png">
